### DISCUSSION TITLE
[BUG] Current skywalking-nginx-lua startBackendTimer cant't support grpc

### DISCUSSION THREAD

**Discussion by weixing-qiao** [apache/skywalking#9132] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

Nginx Lua Agent (apache/skywalking-nginx-lua)

### What happened

I want to collect nginx data by Satellite. as we know Satellite provides 11800 grpc port. I find that skywalking-nginx-lua can't support grpc.  Could we any good idea for it ? 

Or can I continue to collect data using OAP?

### What you expected to happen

Could we any good idea for it ? 

### How to reproduce

none

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment, marked as answer]
I am afraid this is not what Satellite is designed for. Satellite is mostly for gRPC based channels, and as Nginx/APISIX/Kong only support HTTP1 to report data, you should make it to OAP directly. Or, you could put another HTTP proxy(Nginx/Envoy etc.) to provide load balance.

You should submit this to discussion because this is not a bug.

**Reply by weixing-qiao** [reply]
Got it, Thanks for your reply.
